templates/strings/objc-h.stencil

```diff
@@ -0,0 +1,60 @@
+// Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
+
+{% if tables.count > 0 %}
+@import Foundation;
```

Comment:
Might be safer to use `#import <Foundation/Foundation.h>` for users that have modules disabled (Obj-C++).

Reply:
good point, thanks.